harness/propertyHelper.js

```diff
@@ -1,5 +1,6 @@
 
 function isConfigurable(obj, name) {
+    var desc = Object.getOwnPropertyDescriptor(obj, name);
```

Comment:
Right -- there is no actual implementation of test isolation or test262 integration yet.  In the interim, I've been identifying tests which mutate the context object and/or trying to be more gentle about it, so we can run as many of the tests as possible in more-standard test environments.

It wouldn't be a bad idea to add a `destructive` (or `nondestructive`) property tag to tests which  stomp on the global state (or which are asserted not to do so).

WRT whether *all* the property tests should be destructive -- IMO this actually greatly complicates the writing of tests.  You have to carefully order your tests so as not to shoot yourself in the foot.  The reason I proposed this PR is that I think it's better to support the test authors by not stomping on the environment unless absolutely necessary, rather than leave around unexploded ordinance for them to stumble into.